IND/InterestRates/MarketDataProviders.rdf

```diff
@@ -0,0 +1,246 @@
+		<fibo-fbc-fct-breg:hasPostalCode>10041</fibo-fbc-fct-breg:hasPostalCode>
+		<fibo-fbc-fct-breg:hasSubdivision rdf:resource="&lcc-3166-2-us;NewYork"/>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-ind-ir-mdp;BGCPartnersIncLegalEntityIdentifierRegistryEntry">
+		<rdf:type rdf:resource="&fibo-fbc-fct-breg;LegalEntityIdentifierRegistryEntry"/>
+		<rdfs:label>BGC Partners, Inc. legal entity identifier registry entry</rdfs:label>
+		<skos:definition>legal entity identifier registry entry for BGC Partners, Inc.</skos:definition>
+		<fibo-fbc-fct-breg:hasInitialRegistrationDate rdf:datatype="&xsd;dateTime">2012-06-06T08:55:00-07:00</fibo-fbc-fct-breg:hasInitialRegistrationDate>
+		<fibo-fbc-fct-breg:hasRegistrationStatus rdf:resource="&fibo-fbc-fct-breg;LapsedStatus"/>
+		<fibo-fbc-fct-breg:hasRegistrationStatusRevisionDate rdf:datatype="&xsd;dateTime">2018-12-14T02:14:00-08:00</fibo-fbc-fct-breg:hasRegistrationStatusRevisionDate>
+		<fibo-fbc-fct-breg:hasRenewalDate rdf:datatype="&xsd;dateTime">2019-12-13T16:31:00-08:00</fibo-fbc-fct-breg:hasRenewalDate>
+		<fibo-fbc-fct-breg:hasValidationLevel rdf:resource="&fibo-fbc-fct-breg;EntityValidationLevelFullyCorroborated"/>
+		<fibo-fnd-rel-rel:comprises rdf:resource="https://www.gleif.org/lei/TF1LXM1YNB81WKUH5G19"/>
+		<fibo-fnd-rel-rel:isIncludedIn rdf:resource="&fibo-fbc-fct-usfsind;GlobalMarketsEntityIdentifierRegistry"/>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-ind-ir-mdp;EuropeanMoneyMarketsInstituteBenchmarkPublisher">
+		<rdf:type rdf:resource="&fibo-ind-ir-ir;InterestRateAuthority"/>
+		<rdfs:label>European Money Markets Institute (EMMI) benchmark publisher</rdfs:label>
+		<skos:definition>individual representing the European Money Markets Institute (EMMI) functional entity that is an international financial information publisher, responsible for the publication of euro-based benchmarks, including Euribor</skos:definition>
+		<fibo-fnd-rel-rel:hasIdentity rdf:resource="&fibo-fbc-fct-eufseind;EuropeanMoneyMarketsInstitute"/>
+		<fibo-fnd-utl-av:adaptedFrom rdf:datatype="&xsd;anyURI">http://www.emmi-benchmarks.eu/</fibo-fnd-utl-av:adaptedFrom>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-ind-ir-mdp;FederalReserveBoardH.15RateResetTimeOfDay">
+		<rdf:type rdf:resource="&fibo-fnd-dt-fd;TimeOfDay"/>
+		<rdfs:label>Federal Reserve Board H.15 rate reset time of day</rdfs:label>
+		<skos:definition>individual representing the time of day that the Federal Reserve Board publishes Selected Interest Rates (Daily) in Schedule H.15</skos:definition>
+		<fibo-fnd-dt-fd:hasTimeValue>T16:15:00-05:00</fibo-fnd-dt-fd:hasTimeValue>
+		<fibo-fnd-utl-av:adaptedFrom rdf:datatype="&xsd;anyURI">https://www.federalreserve.gov/releases/h15/</fibo-fnd-utl-av:adaptedFrom>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-ind-ir-mdp;FenicsMarketData">
+		<rdf:type rdf:resource="&fibo-fbc-pas-fpas;FinancialServiceProvider"/>
```

Comment:
Provision of MarketData is a distinct role from provision of FinancialServices and so should be a separate individual, based on the logic that we seem to be using..

Reply:
They do both, but I removed it, since in this case it is not important to what we are claiming.